Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -351,15 +351,16 @@ class FirefoxHomeViewController: UICollectionViewController, HomePanel, FeatureF
             self.updateTreesCell()
         }
         
-        if Referrals.isActive {
-            referrals.startSilentRefresh()
-        }
     }
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         reloadAll()
 
+        if Referrals.isActive {
+            referrals.refresh()
```

Comment:
@vauxhall here we do the "regular" refresh which will be throttled by 1 hour logic